docs/source/en/conceptual/philosophy.mdx

```diff
@@ -12,6 +12,99 @@ specific language governing permissions and limitations under the License.
+🧨 Diffusers provides **state-of-the-art** pretrained diffusion models across multiple modalities.
+Its purpose is to serve as a **modular toolbox** for both inference and training.
+
+We aim at building a library that stands the test of time and therefore take API design very seriously.
+
+In a nutshell, Diffusers is built to be a natural extension of PyTorch. Therefore, most of our design choices are based on [PyTorch's Design Principles](https://pytorch.org/docs/stable/community/design.html#pytorch-design-philosophy). Let's go over the most important ones:
+
+## Usability over Performance
+
+- While Diffusers has many built-in performance-enhancing features (see [Memory and Speed](https://huggingface.co/docs/diffusers/optimization/fp16)), models are always loaded with the highest precision and lowest optimization. Therefore, by default diffusion pipelines are always instantiated on CPU with float32 precision if not otherwise defined by the user. This ensures usability across different platforms and accelerators and means that no complex installations are required to run the library.
+- Diffusers aim at being a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
+- Diffusers prefers simple, self-explainable code over condensed, magic code. This means that short-hand code syntaxes such as lemma functions, and advanced PyTorch operators are often not desired.
+
+## Simple over easy
+
+As PyTorch states, **explicit is better than implicit** and **simple is better than complex**. This design philosophy is reflected in multiple parts of the library: 
+- We follow PyTorch's API with methods like [~`DiffusionPipeline.to`] to let the user handle device management.
+- Raising concise error messages is preferred to silently correct erroneous input. Diffusers aims at teaching the user, rather than making the library as easy to use as possible.
+- Complex model vs. scheduler logic is exposed instead of magically handled inside. Schedulers/Samplers are separated from diffusion models with minimal dependencies on each other. This forces the user to write the unroll the denoising loop, but allows for easier debugging, and gives the user more control over adapting the denoising process or switching out diffusion models or schedulers.
+- Separately trained model weights of the diffusion pipeline, *e.g.* the text encoder, the unet, and the variational auto encoder, are separated into different components instead of merged into a single model class. This forces the user to take care of the interaction between the different models making the serialization format more modularized but again allowing for easier debugging and customization. Dreambooth or textual inversion training 
```

Comment:
```suggestion
- Separately trained model weights of the diffusion pipeline, *e.g.* the text encoder, the unet, and the variational auto encoder, each have their own model class. This forces the user to handle the interaction between the different model components, and the serialization format separates the model components into different files. However, this allows for easier debugging and customization. Dreambooth or textual inversion training 
```

"are separated into different components instead of merged into a single model class" -> Can state a bit simpler with "each have their own model class" implying they're separated into different components

"between the different models making the serialization forma" -> the gerund phrase is a bit of a run on imo

"more modularized" -> I think it might be a bit ambiguous to newcomers what modular means in this context